install/terraform/modules/aks/aks.tf

```diff
@@ -60,7 +60,7 @@ resource "azurerm_kubernetes_cluster" "agones" {
   resource_group_name = azurerm_resource_group.agones_rg.name
   dns_prefix          = "agones"
 
-  kubernetes_version = "1.16.13"
+  kubernetes_version = "1.17.13"
```

Comment:
Using `sudo docker run -it mcr.microsoft.com/azure-cli` I verified that this minor version is available:

```
bash-5.0# az aks get-versions --location eastus --output table
KubernetesVersion    Upgrades
-------------------  -------------------------
1.19.3               None available
1.19.1               1.19.3
1.18.10              1.19.1, 1.19.3
1.18.8               1.18.10, 1.19.1, 1.19.3
1.17.13              1.18.8, 1.18.10
1.17.11              1.17.13, 1.18.8, 1.18.10
1.16.15              1.17.11, 1.17.13
1.16.13              1.16.15, 1.17.11, 1.17.13
```

Reply:
I checked our last community meeting notes where you did the same :smile:  but thanks for the confirm!